theano/scalar/basic.py

```diff
@@ -604,12 +604,11 @@ def __sub__(self, other):
     def __mul__(self, other):
         return mul(self, other)
 
-    if PY3:
-        def __truediv__(self, other):
-            return div_proxy(self, other)
-    else:
-        def __div__(self, other):
-            return div_proxy(self, other)
```

Comment:
I think we need to add 2 tests file:
- theano/scalar/tests/test_div_no_futur.py
- theano/scalar/tests/test_div_futur.py

They should be mostly the same, except that one do the import about futur. They should check that we do the correct choise for the give python that is currently running.

This way, we will be sure we do it correctly and that it don't get broken in the futur.